content/en/docs/distributions/charmed/_index.md

```diff
@@ -1,14 +1,30 @@
 +++
-title = "Charmed Kubeflow"
-description = "A free, enterprise-ready Kubeflow distribution from Canonical – with 24/7 support option, expert professional services on-hand, and managed service option also available"
+title = "Charmed Kubeflow from Canonical"
+description = "A production-ready, free-to-use, Kubeflow distribution from Canonical for easy consumption anywhere, from workstations to on-prem, public cloud and edge."
 weight = 50
 +++
 
 
-[Charmed Kubeflow](https://charmed-kubeflow.io/) from [Canonical](https://www.canonical.com) delivers a powerful, sophisticated end-to-end MLOps platform which you can deploy in half an hour or less using any conformant Kubernetes distribution. The enterprise-ready solution is available backed with 24/7 support, expert set-up services, and managed services with an SLA. With a growing ecosystem of powerful extensions and integrations including MLFlow, the Charmed Kubeflow solution can dramatically amplify productivity levels for data scientists and engineers working with advanced analytics and AI.
+[Charmed Kubeflow](https://charmed-kubeflow.io/) from [Canonical](https://www.canonical.com/) delivers a powerful, sophisticated end-to-end MLOps platform which you can deploy using any conformant Kubernetes distribution. The solution enables optimised AI training and modelling, in a robust and automated way, allowing data scientists to focus on AI/ML projects, not tools. The enterprise-ready platform is available backed with 24/7 support, expert set-up services, and managed services with a service level agreement (SLA). With a growing ecosystem of powerful extensions and integrations including MLFlow or Seldon, the Charmed Kubeflow solution amplifies productivity levels for data scientists and machine learning engineers working with advanced analytics and AI.
```

Comment:
```suggestion
[Charmed Kubeflow](https://charmed-kubeflow.io/) from [Canonical](https://www.canonical.com/) delivers a powerful, sophisticated end-to-end MLOps platform which you can deploy using any conformant Kubernetes distribution. The solution enables optimised AI training and modelling, in a robust and automated way, allowing data scientists to focus on AI/ML projects, instead of the underlying infrastructure. The enterprise-ready platform is available backed with 24/7 support, expert set-up services, and managed services with a service level agreement (SLA). With a growing ecosystem of powerful extensions and integrations including MLFlow and Seldon, the Charmed Kubeflow solution amplifies productivity levels for data scientists and machine learning engineers working with advanced analytics and AI.
```